### ISSUE TITLE
ngx-lite/modal closes on any keyup event

### ISSUE THREAD

**Issue by mickymcq** [coryrylan/ngx-lite#70]
**Describe the bug**
ngx-lite/modal closes on any keyup event now (not just the escape key). I think it was in the PR that updated to Angular 11. I believe the following screenshot shows the code that made it start doing that.

**Screenshots**
![image](https://user-images.githubusercontent.com/11476652/158908530-f664ab84-835d-4ec7-b3ec-6c4cf4ee6429.png)

**Comment by KMK0815** [comment]
Hello,
how can i avoid this?

**Comment by pranavirabatti** [comment]
Yes, I don't know why it was changed. I can help you to fix this. 
@coryrylan
